app/assets/stylesheets/searchworks4/popovers.css

```diff
@@ -0,0 +1,5 @@
+.popover-availability {
```

Comment:
since this is just one rule, can this move into `searchworks4/record.css`?  More css files mean more loads, so we don't want to add them without some consideration.

Reply:
Yes, done